background.ts

```diff
@@ -569,31 +570,25 @@ mainEvents.on('integration-update', (state) => {
  * cluster (if any), and delete all of its data.  This will also remove any
  * rancher-desktop data, and restart the application.
  */
-async function doFactoryReset(keepSystemImages: boolean) {
-  await k8smanager.factoryReset(keepSystemImages);
-  await pathManager.remove();
-  await integrationManager.remove();
-  if (os.platform() === 'win32') {
-    // On Windows, we need to use a helper process in order to ensure we
-    // delete files in use.  Of course, we can't wait for that process to
-    // return - the whole point is for us to not be running.
-    childProcess.spawn(path.join(paths.resources, 'win32', 'wsl-helper.exe'),
-      ['factory-reset', `--wait-pid=${ process.pid }`, `--keep-system-images=${ keepSystemImages ? 'true' : 'false' }`],
-      { detached: true, windowsHide: true });
-    Electron.app.quit();
+function doFactoryReset(keepSystemImages: boolean) {
+  // Don't wait for this process to return -- the whole point is for us to not be running.
+  const tmpdir = os.tmpdir();
+  const outfile = fs.openSync(path.join(tmpdir, 'rdctl-stdout.txt'), 'w');
+  const errfile = fs.openSync(path.join(tmpdir, 'rdctl-stderr.txt'), 'w');
```

Comment:
I made the function async (again), am just using the one FD, and wrote up https://github.com/rancher-sandbox/docs.rancherdesktop.io/issues/116 as a doc bug because the user is only going to see that message if they're tailing `background.log`.